packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -546,96 +547,7 @@ export function isNetwork(chainId: ChainId) {
 }
 
 export function getNetworkName(chainId: number) {
```

Comment:
This is used in a lot of places and we are familiar with this method. We also use it quite often, might be better to leave it wrapped as is